audit/projects/k8s-release/buckets/k8s-release/iam.json

```diff
@@ -25,7 +26,8 @@
       "members": [
         "group:k8s-infra-artifact-admins@kubernetes.io",
         "group:k8s-infra-release-admins@kubernetes.io",
-        "group:k8s-infra-release-editors@kubernetes.io"
+        "group:k8s-infra-release-editors@kubernetes.io",
+        "serviceAccount:project-304687256732@storage-transfer-service.iam.gserviceaccount.com"
```

Comment:
Same thing, but now I'm a little more annoyed, since this is a higher-blast-radius role and I didn't setup a job that required all of it.

https://cloud.google.com/storage-transfer/docs/iam-transfer#sink-permissions says we could use `roles/storage.legacyBucketWriter` instead, but I'm trying to avoid that role (ref: https://github.com/kubernetes/k8s.io/issues/2007)

Reply:
```
gsutil iam ch -d serviceAccount:project-304687256732@storage-transfer-service.iam.gserviceaccount.com:roles/storage.objectAdmin gs://k8s-release
```